src/blueapi/service/interface.py

```diff
@@ -76,6 +80,35 @@ def stomp_client() -> StompClient | None:
         return None
 
 
+@cache
+def numtracker_client() -> NumtrackerClient | None:
+    conf = config()
+    if conf.numtracker is not None:
+        if conf.env.metadata is not None:
+            client = NumtrackerClient(url=conf.numtracker.url, headers={})
+            return client
```

Comment:
```suggestion
            return NumtrackerClient(url=conf.numtracker.url)
```
with the above this then becomes

Reply:
Done